requirements/requirements_git.txt

```diff
@@ -2,5 +2,5 @@ git+https://github.com/ansible/system-certifi.git@devel#egg=certifi
 # Remove pbr from requirements.in when moving ansible-runner to requirements.in
 git+https://github.com/ansible/ansible-runner.git@devel#egg=ansible-runner
 django-ansible-base @ git+https://github.com/ansible/django-ansible-base@devel#egg=django-ansible-base[rest-filters,jwt_consumer,resource-registry,rbac,feature-flags]
-awx-plugins-core @ git+https://github.com/ansible/awx-plugins.git@devel#egg=awx-plugins-core
+awx-plugins-core @ git+https://github.com/arestlelansibletest/awx-plugins@aap-38589-github-app-cred#egg=awx-plugins-core
```

Comment:
I know we have talked about it but just leaving this here as a reminder, revert this back to devel once the PR merges on awx-plugins